test/helper.rb

```diff
@@ -159,6 +159,10 @@ def target_version(target)
       end
     end
 
+    def omit_version(min_ver)
```

Comment:
This isn't used anywhere. Can you remove it from the PR please?

Reply:
The above helper method is used by the following linter.
https://github.com/redis/redis-rb/pull/799/files#diff-a14ac8847c163e36437c11a9aa3c7643R8-R11

Reply:
Weird, that's a broken link for me and `ctrl + F omit_version` doesn't match anything.

Reply:
Argh, nvm, GitHub chose to hide that part of the diff.

Reply:
I'm sorry.

```ruby
def setup
  super
  omit_version(MIN_REDIS_VERSION)
end
```